src/components/Map/index.tsx

```diff
@@ -0,0 +1,119 @@
+import { BikeRentalStation, Scooter } from '@entur/sdk'
+import React, { useState } from 'react'
+
+import ReactMapGL, { Marker } from 'react-map-gl'
+
+import PositionPin from '../../assets/icons/positionPin'
+import ScooterOperatorLogo from '../../assets/icons/scooterOperatorLogo'
+
+import { DEFAULT_ZOOM } from '../../constants'
+import { useSettingsContext } from '../../settings'
+import { StopPlaceWithDepartures } from '../../types'
+
+import BikeRentalStationTag from './BikeRentalStationTag'
+import StopPlaceTag from './StopPlaceTag'
+
+import './styles.scss'
+
+const MapView = ({
```

Comment:
Denne vil du kanskje kalle kun `Map`, for `MapView` er vel kartvisningstypen?

Reply:
Trodde Map var beskyttet, men ser ut til å fungere